### ISSUE TITLE
add about screen

### ISSUE THREAD

**Issue by mkinney** [meshtastic/Meshtastic-gui-installer#18]
(make sure the menu bar does not say "Python")

**Comment by fsgl1tch** [comment]
What ought to be added to the about page? Additional button on the form? New closable window for the about page? Clicking the logo redirects to homepage, move to about page?

**Comment by mkinney** [comment]
About page could be whatever, link to GitHub and Meshtastic website.

Clicking the "X" in the top corner should suffice to close it.

Moving the logo to redirect would be good to move (or copy?) there. I have to admit, that I've clicked on that logo a few times during testing accidentally.

I had hoped to "hide" the Python in the top left corner (on Mac) when m-flasher was running, but I don't think that is an easy feat. I thought if we added an "About" page (on "File" menu) that the Python would go away. I don't think that's the case anymore. For that "Python" to no longer show, at least on mac, there'd have to be an Info.plist with icon bundles and whatnot.
